src/lib/y2partitioner/widgets/configure.rb

```diff
@@ -202,6 +204,10 @@ def initialize(id, label, icon, client, pkgs)
         # are indexed in this constant in order to reuse the existing
         # translations from yast2-storage
         WARNING_TEXTS = {
+          crypt: N_(
+            "Rescanning crypt devices cancels all current changes.\n" \
```

Comment:
NP: "discards" sounds better for me then "cancels"

Reply:
Intentionally done for symmetry with all the other texts (that were probably inherited from the old storage).